docs/spec-files/message_structure.md

```diff
@@ -88,6 +88,17 @@ Headers in [DIDComm Messaging](.) are intended to be extensible in much the same
 - **typ** - OPTIONAL. Media type of the JWM content.
 
 - **to** - OPTIONAL. Identifier(s) for recipients. MUST be an array of strings where each element is a valid [DID](https://www.w3.org/TR/did-core/) or [DID URL](https://w3c.github.io/did-core/#did-url-syntax) (without the [fragment component](https://w3c.github.io/did-core/#fragment)) that identifies a member of the message's intended audience.
```

Comment:
Notice that this PR actually duplicated the **to** field entry, letting the old one above the new part about the table for the media types.